.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+What does this PR do?
```

Comment:
This seems a bit overkill for this repo.
The `checklist` does not really match what we do here.
I think What, Why, Impacted Scenarios is enough

Reply:
Done !